frontend/src/pages/Signin.tsx

```diff
@@ -27,9 +29,15 @@ function Signin() {
     try {
       setErrorMessage(undefined);
       setSubmitStatus("sending");
-      const authToken = await backendClient.signinUser(username, password);
-      SessionStorage.storeAccessToken(authToken);
-      setSubmitStatus("success");
+      // Disallow signing with demo credents if not on demo mode
```

Comment:
why we care if someone will login with this credentials?

Reply:
Yinon asked to restrict the usage of demo account only if on demo mode. One of the reasons why, I believe, is because we'll keep the demo account funded at all times (with testnet coins) and we want to prevent abuse. But better discuss with Yinon for the full explanation.